Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -29,8 +63,19 @@ public static void Reverse(int[] array)
         /// <returns>A new array with the last element removed. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveLast(int[] array)
```

Comment:
What does RemoveLast and RemoveAt have in common :) ?

Reply:
NullOrEmptyArray(array) function